pkg/cmd/render/render_test.go

```diff
@@ -593,3 +671,61 @@ func Test_getMachineCIDR(t *testing.T) {
 		})
 	}
 }
+
+const installConfigSingleStackIPv6DelayedHA = `apiVersion: v1
+metadata:
+  name: my-cluster
+networking:
+  clusterNetwork:
+  - cidr: 10.128.0.0/14
+    hostPrefix: 23
+  machineCIDR: 10.0.0.0/16
+  machineNetwork:
+  - foo: bar
+    cidr: 2620:52:0:1302::/64
+  networkType: OpenShiftSDN
+  serviceNetwork:
+  - 172.30.0.0/16
+bootstrapInPlace:
+      installationDisk: /dev/sda
+`
+
+func Test_getBootstrapScalingStrategy(t *testing.T) {
```

Comment:
(No action required) Given the simplicity of the function under test - 2 conditionals and a return statement constituting 5 lines of code - I'm not sure this level of testing is suggested.